src/firefly/js/metaConvert/PartAnalyzer.js

```diff
@@ -71,7 +73,8 @@ export function chooseDefaultEntry(menu,parts,fileFormat, dataTypeHint) {
 function findAvailableTypesForAnalysisPart(part, fileFormat) {
     const {type}= part;
     if (type===FileAnalysisType.HeaderOnly || type===FileAnalysisType.Unknown) return [];
-    if (type!==FileAnalysisType.Image || fileFormat!=='FITS' || is1DImage(part)) return [DPtypes.CHART,DPtypes.TABLE];
+
+    if (type!==FileAnalysisType.Image &&  fileFormat!=='FITS' &&  is1DImage(part)) return [DPtypes.CHART,DPtypes.TABLE];
```

Comment:
I see how this logic is very confusing, I was confused myself until I did some testing.  The fix is not correct, it breaks out reading VO tables (and maybe other tables). We should talk about what the right fix is.
Probably just having an if right before.
```
if (type===FileAnalysisType.Table) return [DPtypes.CHART,DPtypes.TABLE];
```
We need to test that though.

Reply:
Can you please let  me know a vo-table test case?  If so, I can figure out a solution.  Thanks!

Reply:
Any changes to the core of the data products viewer should be tested with a firefly build using the file:  http://web.ipac.caltech.edu.s3-us-west-2.amazonaws.com/staff/roby/data-products-test/dp-test.tbl

Reply:
I guess if I merge my branch, I can test the vo-table.  Let me do that first.

Reply:
I tried my suggested fix and it works for dp-test.tbl. Maybe it fixed the GREAT problem as well.

Reply:
> Any changes to the core of the data products viewer should be tested with a firefly build using the file: http://web.ipac.caltech.edu.s3-us-west-2.amazonaws.com/staff/roby/data-products-test/dp-test.tbl

Good to know.  I will remember that from now on.